src/containers/Transactions/test/Description.test.js

```diff
@@ -52,7 +52,7 @@ describe('Description container', () => {
   it('renders description for Payment', () => {
     const wrapper = createWrapper(Payment)
     expect(wrapper.html()).toBe(
-      '<div class="section"><div class="title">description</div><div>transaction_sequence<b> <span>31030</span></b></div>The payment is from<a class="account" title="rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok" href="/accounts/rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok">rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok</a>to<a class="account" title="rHoPwMC75KVUhBMeV3uDMybKG5JND74teh" href="/accounts/rHoPwMC75KVUhBMeV3uDMybKG5JND74teh">rHoPwMC75KVUhBMeV3uDMybKG5JND74teh</a><div>the_source_tag_is<b> SourceTag</b></div><div>the_destination_tag_is<b> DestinationTag</b></div><div>payment_desc_line_4<b><span> 2,421.8268</span><small>XRP</small></b></div><div>payment_desc_line_6<b><span> 2,421.8268</span><small>XRP</small></b></div></div>',
+      '<div class="section"><div class="title">description</div><div>transaction_sequence<b> <span>31030</span></b></div>The payment is from<a class="account" title="rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok" href="/accounts/rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok">rNQEMJA4PsoSrZRn9J6RajAYhcDzzhf8ok</a>to<a class="account" title="rHoPwMC75KVUhBMeV3uDMybKG5JND74teh" href="/accounts/rHoPwMC75KVUhBMeV3uDMybKG5JND74teh">rHoPwMC75KVUhBMeV3uDMybKG5JND74teh</a><div>the_source_tag_is<b> SourceTag</b></div><div>the_destination_tag_is<b> DestinationTag</b></div><div>payment_desc_line_4<b><span> \uE9002,421.8268</span><small>XRP</small></b></div><div>payment_desc_line_6<b><span> \uE9002,421.8268</span><small>XRP</small></b></div></div>',
```

Comment:
Did these suddenly break?  This should be its own PR.

Reply:
It's just editing the box to the actual XRP code

Reply:
Yeah i get that.  It is just tangentially related to the rest of the PR.

Reply:
Yeah, that's fair. If you want me to move it to a separate PR I can do that, it just seemed like a pretty small change.